governator-test-junit/src/main/java/com/netflix/governator/guice/test/TestInjectorCreationMode.java

```diff
@@ -0,0 +1,6 @@
+package com.netflix.governator.guice.test;
+
+public enum TestInjectorCreationMode {
+    BEFORE_CLASS,
```

Comment:
I'd probably match the JUnit style of enum values here: { BEFORE_CLASS, BEFORE } is a match for existing annotations @Before and @BeforeClass